util/collections.go

```diff
@@ -49,3 +54,12 @@ func removeDuplicatesFromList(list []string, keepLast bool) []string {
 	}
 	return out
 }
+
+// Returns an HCL compliant formated list of strings
+func ListToHCLArray(list []string) string {
```

Comment:
Initially, this function was returning a list ;-) but you preferred that the user add the bracket itself.